.github/workflows/dusk_ci.yml

```diff
@@ -41,7 +42,6 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2020-10-25
           override: true
```

Comment:
Nit: this also needs to be removed (see again: https://github.com/dusk-network/rusk/pull/320/files as reference), you don't have anything to override anymore.